R/create.R

```diff
@@ -347,20 +351,22 @@ create_gp_data <- function(gp = gp_opts(), data) {
 
   # map settings to underlying gp stan requirements
   gp_data <- list(
-    fixed = as.numeric(fixed),
-    M = M,
-    L = gp$boundary_scale,
-    ls_meanlog = convert_to_logmean(gp$ls_mean, gp$ls_sd),
-    ls_sdlog = convert_to_logsd(gp$ls_mean, gp$ls_sd),
-    ls_min = gp$ls_min,
-    ls_max = data$t - data$seeding_time - data$horizon,
-    alpha_sd = gp$alpha_sd,
-    # nolint start
-    gp_type = ifelse(gp$kernel == "se", 0,
-      ifelse(gp$kernel == "matern", 1, 0)
+  fixed = as.numeric(fixed),
```

Comment:
are these indented correctly? It looks like it has changed but perhaps that is just an artefact?

Reply:
Looks alright on my end.

Reply:
why are they showing up as changes? Were they not correctly indented before?

Reply:
I styled each part where I made a change, so that's probably why.